.pipelines/pipeline.yaml

```diff
@@ -388,7 +388,7 @@ stages:
       os: linux
       clusterType: dualstack-overlay-byocni-up
       clusterName: "dualstackoverlaye2e"
-      vmSize: Standard_B2s
+      vmSize: Standard_D2a_v4
```

Comment:
Why are we increasing the size ?

Reply:
(1) We need to have a better sku to test because sometimes you will hit this issue:
```
WARNING: The behavior of this command has been altered by the following extension: aks-preview
ERROR: (CreateVMSSAgentPoolFailed) Code="OverconstrainedAllocationRequest" Message="Allocation failed. VM(s) with the following constraints cannot be allocated, because the condition is too restrictive. Please remove some constraints and try again. Constraints applied are:\n  - Networking Constraints (such as Accelerated Networking or IPv6)\n  - VM Size\n" Target="2"
Code: CreateVMSSAgentPoolFailed

```
(2) When HNS team checked an issue before. With Standard_B2s sku, there is even no memory to copy logs out from the node. So it's easier for debug.